torch/csrc/jit/frontend/script_type_parser.cpp

```diff
@@ -217,13 +217,81 @@ TypePtr ScriptTypeParser::parseType(const std::string& str) {
   return parseTypeFromExpr(p.parseExp());
 }
 
+bool checkMutableFunctionDefault(const IValue& def) {
+  // TODO: Renable.
```

Comment:
I dont think we should disable this to land the PR - can you comment why this was necessary?

Reply:
That was not my intention - I'm still trying to wade through a bunch of broken tests to figure out what's going wrong with trying to parse default arguments everywhere (not just in class types). I have not yet requested review for this reason.

Reply:
I want to reenable this, but in the process of debugging some broken tests, I found some that somehow do have list-literals as default arguments, which are considered mutable. I was not able to find them in the codebase anywhere, so I think they must be generated somehow (perhaps with `native_functions.yaml`). I was going to bring this up after sifting through all of the tests to figure out why parsing defaults everywhere causes so many problems.